### PR TITLE
Fix handling of absolute URLs in StreamResolver

### PR DESCRIPTION
**Pull request by justinfagnani**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
   "author": "The Polymer Project Authors",
   "license": "BSD-3-Clause",
   "dependencies": {
+    "chai": "^3.5.0",
```
Comment (azakus) [inline package.json]: This should be a devDependency right?

Comment (justinfagnani) [inline package.json]: looks like you already fixed

**Comment by justinfagnani** [comment]
@azakus added a test. PTAL

**Comment by azakus** [comment]
LGTM

**Pull request merged**